components/admin/data/widgets/pages/show/component.js

```diff
@@ -18,9 +18,9 @@ class WidgetsShow extends PureComponent {
   handleSubmit = (widget) => {
     if (widget) {
       Router.pushRoute('admin_data_detail', {
```

Comment:
Aren't we just trying to go to the same page after updating a widget? I think we could do nothing for that case.

Reply:
@andresgnlez I've just updated the code so that the page scrolls to the top instead of reloading it.
Should we merge it now?